### DISCUSSION TITLE
Selenium Docker Extension

### DISCUSSION THREAD

**Discussion by ivan-curkovic** [docker/extension-ideas#165] [Ideas]
Selenium Docker images are quite popular. 
🔗Selenium on Docker Hub - https://hub.docker.com/u/selenium
🏠Selenium Home - https://www.selenium.dev/
🧑‍💻Selenium GitHub - https://github.com/SeleniumHQ/docker-selenium

What would be useful integration of Docker and Selenium?

**Comment by ajeetraina** [comment]
Built a Selenium Docker extension https://github.com/collabnix/selenium-docker-extension successfully.
![image](https://github.com/docker/extension-ideas/assets/313480/95fc85c3-2159-4dcc-b358-bc84e6218bf6)

Give it a try!
